lib/core/test/integration/Test/Integration/Scenario/API/Transactions.hs

```diff
@@ -568,7 +568,7 @@ spec = do
         expectErrorMessage (errMsg404NoWallet $ w ^. walletId) r
 
     describe "TRANS_CREATE_08 - v2/wallets/{id}/transactions - Methods Not Allowed" $ do
-        let matrix = ["PUT", "DELETE", "CONNECT", "TRACE", "OPTIONS", "GET"]
+        let matrix = ["PUT", "DELETE", "CONNECT", "TRACE", "OPTIONS"]
```

Comment:
Had to remove this as we now have implemented `GET v2/wallets/{id}/transactions`.

Not sure if we should add a new test case for `ListTransactions` now or not.